client/components/bancontactPayments/html/README.md

```diff
@@ -0,0 +1,170 @@
+   - Select **Payment methods** from the left sidebar
+   - Find **Bancontact** in the payment methods and enable it
+
+### 2. System Requirements
+
+- Node.js version 20 or higher
+- Server running on port 8080 (see `server/node/` directory)
+
+## 🚀 Running the Demo
+
+### Server Setup
+
+1. **Navigate to the server directory:**
+
+   ```bash
+   cd server/node
+   ```
+
+2. **Install dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Configure environment variables:**
+   Create a `.env` file in the root directory using your client credentials from the previous Create Application step:
+
+   ```env
+   PAYPAL_SANDBOX_CLIENT_ID=your_paypal_sandbox_client_id
+   PAYPAL_SANDBOX_CLIENT_SECRET=your_paypal_sandbox_client_secret
+   ```
+
+4. **Start the server:**
+   ```bash
+   npm start
+   ```
+   The server will run on `http://localhost:8080`
+
+### Client Setup
+
+1. **Navigate to the Bancontact demo directory:**
+
+   ```bash
+   cd client/components/bancontactPayments/html
+   ```
+
+2. **Install dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Start the development server:**
+   ```bash
+   npm start
+   ```
+   The demo will be available at `http://localhost:3000`
+
+## How It Works
+
+### Client-Side Flow
+
+1. **SDK Initialization**: Loads PayPal Web SDK with Bancontact components using a client token fetched from the server's `/paypal-api/auth/browser-safe-client-token` endpoint
+2. **Eligibility Check**: Verifies Bancontact is eligible for the merchant
+3. **Session Creation**: Creates Bancontact payment session with event callbacks for handling payment lifecycle events
+4. **Field Setup**: Mounts the required full name field
+5. **Validation**: Validates fields before initiating the payment flow
+6. **Payment Flow**: Opens a popup window where customers select their bank and authorize the payment. The order is created server-side via `/paypal-api/checkout/orders/create` before displaying the popup
+7. **Completion**: Processes the payment result by capturing the approved order via `/paypal-api/checkout/orders/:orderId/capture`, or handles cancellation and error scenarios appropriately
+
+### Server-Side Requirements
+
+The integration requires these endpoints (provided by the API server):
+
+- `GET /paypal-api/auth/browser-safe-client-token` - Generate client token
+- `POST /paypal-api/checkout/orders/create` - Create order
+- `POST /paypal-api/checkout/orders/:orderId/capture` - Capture order
+
+## Available Scripts
+
+- `npm start` - Start Vite development server (port 3000)
+- `npm run build` - Build for production
+- `npm run preview` - Preview production build
+- `npm run format` - Format code with Prettier
+- `npm run format:check` - Check code formatting
+
+## Troubleshooting
+
+### Bancontact not eligible
+
+- Verify `testBuyerCountry` is set to "BE"
+- Check that Bancontact is enabled for the merchant in PayPal account settings
+
+### Validation fails
+
+- Ensure full name field is properly mounted
```

Comment:
Would it be helpful to elaborate on "properly mounted"? I think that could mean one of several things.